app/javascript/pages/MyEvents/CreateEditModalContents.jsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useContext } from 'react'
+
+import ReduxFormAutocomplete from 'components/ReduxFormAutoComplete'
+import TagField from 'components/TagField'
+import { UserContext } from '../../context/UserContext'
+import { Header, Body } from '@zendeskgarden/react-modals'
+import { Field as GField, Label, Hint, Input, Message, Range } from '@zendeskgarden/react-forms'
+import { Datepicker } from '@zendeskgarden/react-datepickers'
+import { Dots } from '@zendeskgarden/react-loaders'
+
+import { Button } from '@zendeskgarden/react-buttons'
+import styled from 'styled-components'
+
+const PaddedField = styled(GField)`
+  flex: 1;
+  padding: ${({ theme }) => theme.space.sm} ${({ theme }) => theme.space.xs};
+`
+
+const TopMargin = styled.div`
+  margin-top: ${({ theme }) => theme.space.xs};
+`
+const RightFloatButton = styled(Button)`
+  float: right;
+`
+
+const CreateEditModalContents = ({
+  offices,
+  eventTypes,
+  tags,
+  organizations,
+  createEditIndividualEvent,
+  isNew,
```

Comment:
I feel like if we are going to have this property, we should validate that `modalEventData` is not null and has a nonzero id when `isNew=false`

Because looking at how the data is submitted to the server, a new event will be created in the absence of an id.